### ISSUE TITLE
This amazing work doesn't work on Arch Linux Gnome 44.2

### ISSUE THREAD

**Issue by AliSrc** [yilozt/mutter-rounded#113]
Hi, 

This is an amazing work. I will call it art, but sadly it doesn't work on my system. 

Do you have any plans to get it to work on Arch with;
Kernel: 6.35+ 
Gnome: 44.2+
WM: X11

I was getting error with testing, so I commented out the check method in PKGBUILD, then I get a Conflict error with mutter and mutter-rounded. So I removed mutter with pacman -Rdd mutter and ran the makepkg -si and installed it without any problem but when I logged out and want to log in again there was an error with gnome-shell, it will not work with mutter-rounded longer. 

Please keep this amazing project alive FOREVER! :) 

My Best Regards!

**Comment by SpikeVN** [comment]
Just three days ago, I did the same thing and also got the same error. Don't try to use incompatible mutter version. It breaks libmutter.
Check #103 for more details about updating to higher GNOME versions.
**tldr**: [more updated fork](https://github.com/Spirrwell/mutter-rounded) (only GNOME 43 in June 2023), however, the patching process is very laborious and has to be done again on every update, so the author has likely abandoned the project (**only my assumption, not officially announced**). It is recommended that you use [Blur My Shell](https://extensions.gnome.org/extension/3193/blur-my-shell/) (now have beta support on application blur) and [Rounded Window Corners](https://extensions.gnome.org/extension/5237/rounded-window-corners/) (by this project's author)

**Comment by SpikeVN** [comment]
Just three days ago, I did the same thing and also got the same error. Don't try to use incompatible mutter version. It breaks libmutter.
Check #103 for more details about updating to higher GNOME versions.
**tldr**: [more updated fork](https://github.com/Spirrwell/mutter-rounded) (only GNOME 43 in June 2023), however, the patching process is very laborious and has to be done again on every update, so the author has likely abandoned the project (**only my assumption, not officially announced**). It is recommended that you use [Blur My Shell](https://extensions.gnome.org/extension/3193/blur-my-shell/) (now have beta support on application blur) and [Rounded Window Corners](https://extensions.gnome.org/extension/5237/rounded-window-corners/) (by this project's author)

**Comment by AliSrc** [comment]
> Just three days ago, I did the same thing and also got the same error. Don't try to use incompatible mutter version. It breaks libmutter. Check #103 for more details about updating to higher GNOME versions. **tldr**: [more updated fork](https://github.com/Spirrwell/mutter-rounded) (only GNOME 43 in June 2023), however, the patching process is very laborious and has to be done again on every update, so the author has likely abandoned the project (**only my assumption, not officially announced**). It is recommended that you use [Blur My Shell](https://extensions.gnome.org/extension/3193/blur-my-shell/) (now have beta support on application blur) and [Rounded Window Corners](https://extensions.gnome.org/extension/5237/rounded-window-corners/) (by this project's author)

Thank you for your input. I just toke a look at Blur my Shell which is a really nice extension to blur the login screen. But it's not good to blur and add a transparency to the windows. Maybe it will be better with time.
